elliott/elliottlib/bzutil.py

```diff
@@ -210,6 +230,7 @@ def creation_time_parsed(self):
 class JIRABug(Bug):
     def __init__(self, bug_obj: Issue):
         super().__init__(bug_obj)
+        self.jira_client = None
```

Comment:
Why do we want to add this..?

Reply:
Okay I think I understand the intent. To get access to the custom field info, we need access to jira client. This is fine then